federated_aws_cli/role_picker.py

```diff
@@ -38,6 +39,27 @@ def get_aws_env_variables(credentials):
     return result
 
 
+def get_aws_shared_credentials(credentials, role_arn):
+    # come up with a profile name that won't break AWS
+    result = "[{}]\naws_access_key_id={}\naws_secret_access_key={}\nrole_arn={}\n".format(
```

Comment:
It probably makes sense for portability to use the `awscli` configuration setting methods. This would mean importing the `awscli` module (using the [`configure'](https://github.com/aws/aws-cli/tree/develop/awscli/customizations/configure) customization). This would allow you to do the same kind of thing that you'd do on the command line like
```
aws configure set aws_session_token "foobar" --profile infosec-dev-MyRole
```

as opposed to writing the file by hand.

Reply:
This could also be done after this PR

Reply:
Let's tackle this after the PR, as I'd like to talk to you about how this would work first.  :)